### DISCUSSION TITLE
[Question]: Does the Coral USB Accelerator require updating?

### DISCUSSION THREAD

**Discussion by Lockie85** [blakeblackshear/frigate#11967] [Ask A Question] [answered]
### What is your question:

Hey,

With everything get updates etc. these days I wondered, does the Coral USB Accelerator also require updating now and then so it performs better? I couldn't seem to find anything, but I've always struggled to understand the Coral device.

Many Thanks

**Comment by banthungprong** [comment]
https://github.com/google-coral/webcoral
and there:

> The firmware is usually automatically flashed by [libedgetpu](https://github.com/google-coral/libedgetpu) library when using C++ or Python programs,...

Latest Commit to libedgetpu was about 3 months ago
(https://github.com/google-coral/libedgetpu/commit/e35aed18fea2e2d25d98352e5a5bd357c170bd4d)

**Reply by Lockie85** [reply]
Humm, mine is plugged into my proxmox machine which runs a Home Assistant VM. Not sure if this would cover the automatic updates?

**Reply by NickM-27** [reply, marked as answer]
It's all built in to the container. There is no "update" that you as a user runs on the coral

**Reply by Lockie85** [reply]
Many thanks
